applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_interface_element.cpp

```diff
@@ -1314,18 +1383,20 @@ KRATOS_TEST_CASE_IN_SUITE(LineInterfaceElement_InterpolatesNodalStresses, Kratos
     p_properties->SetValue(YOUNG_MODULUS, 1.000000e+07);
     p_properties->SetValue(POISSON_RATIO, 0.000000e+00);
     // create a triangle neighbour element
-    auto p_neighbour_element = ElementSetupUtilities::Create2D3NElement(nodes, p_properties, 2);
+    auto p_neighbour_element = make_intrusive<MockElementWithTotalStressVectors>(
+        2, std::make_shared<Triangle2D3<Node>>(nodes), p_properties);
+    p_neighbour_element->SetIntegrationMethod(GeometryData::IntegrationMethod::GI_GAUSS_2);
     ProcessInfo dummy_process_info;
     p_neighbour_element->Initialize(dummy_process_info);
-    std::vector<ConstitutiveLaw::StressVectorType> r_stress_vectors;
+    std::vector<ConstitutiveLaw::StressVectorType> total_stress_vectors;
     ConstitutiveLaw::StressVectorType              stress_vector(4);
     stress_vector <<= 3.0, 13.0 / 6.0, 4.0, 1.0;
-    r_stress_vectors.emplace_back(stress_vector);
-    r_stress_vectors.emplace_back(stress_vector);
+    total_stress_vectors.emplace_back(stress_vector);
```

Comment:
Let's not change this now, but for future work, we can use the UblasUtils:

```suggestion
    total_stress_vectors.emplace_back(UblasUtilities::CreateVector({3.0, 13.0 / 6.0, 4.0, 1.0}));
```

Reply:
Yes, I agree. Let's do this in a separate PR. Thanks for your suggestion.